test/functional/utilities/createuvm.go

```diff
@@ -10,7 +10,7 @@ import (
 
```

Comment:
Do you want to keep the changes under test/functional since this suite if out of date and not run anywhere right now? Would they be valuable once we have these tests working again?

Reply:
I'd like to keep them, since they do exercise and demonstrate one of the two new APIs. I believe that I ran these tests locally (by accident, trying to work out the Go syntax to just run *a* test) and the only one that failed was about UVM RAM-management.